tests/rptest/tests/consumer_offsets_migration_test.py

```diff
@@ -0,0 +1,118 @@
+# Copyright 2022 Vectorized, Inc.
```

Comment:
@mmaslankaprv - does ducktape have mm2 support? wondering if we setup an mm2 migration of 10 records with consumer reading 7 of them and ensuring that the groups migrate. could be a separate PR.

Reply:
As we spoke earlier, we will add a MM2 test as a follow up.

Reply:
there should be a requirement to run mirror-maker2 process between 1 apache kafka cluster and 1 redpanda cluster -> that's the MVP kafka to redpanda 0 downtime migration.